README.md

```diff
@@ -95,4 +95,4 @@ It will create a new file called `JMdict.json` in the same directory.
 
 # Download
 
-I have already created one for the English **only** JMdict file. You can grab it from [here](https://droppy.cryf.in/$/qfxsg).
\ No newline at end of file
+I have already created one for the English **only** JMdict file. You can grab it from [here](https://droppy.cryf.in/$/qfxsg).
```

Comment:
What is the change in this line? The GitHub diff checker is not showing any changes.

Reply:
That was an unintended change that happened when I saved the file in vim.
Per default, it adds a newline at the end of the file. I'll commit a fix.

Reply:
Ah I see. Thank you!